spec/rails_helper.rb

```diff
@@ -69,6 +69,7 @@
   end
 
   config.around(:each) do |example|
+    Settings.reload!
```

Comment:
The test runs I did showed no measurable time difference (Down to the millisecond) running 630 specs with or without the reload included.